Ag 3390 grid charts website sync

markdurrant opened the pull request:
Add the following components from Grid. 

- Trial form docs
- Trial form pricing page 
- Quick start "Next" cards
- Com vs Enterprise feature cards

markdurrant committed:
git subrepo pull external/ag-website-shared

subrepo:
  subdir:   "external/ag-website-shared"
  merged:   "c36ff43fde"
upstream:
  origin:   "git@github.com:ag-grid/ag-website-shared.git"
  branch:   "latest"
  commit:   "c36ff43fde"
git-subrepo:
  version:  "0.4.6"
  origin:   "git@github.com:ingydotnet/git-subrepo.git"
  commit:   "73a0129"

markdurrant committed:
AG-3390 - Add trial licences forms

markdurrant committed:
AG-3390 - Add getting started cards component

markdurrant committed:
AG-3390 - Refactor getting started cards component

markdurrant committed:
AG-3390 - Seperate out Grid & Charts getting started cards data

markdurrant committed:
AG-3390 - Add com vs enterprise features cards components

markdurrant committed:
AG-3390 - Refactor docsFeatures component & seperate out data

markdurrant committed:
AG-3390 - Fix linting issue

taktran commented on external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json:
```diff
@@ -0,0 +1,162 @@
+            {
+                "title": "Financial Charts",
+                "description": "Build interactive financial charts with minimal configuration.",
+                "link": "financial-charts/"
```
Needs to be a relative link

```suggestion
                "link": "./financial-charts/"
```

(this is failing the build)

taktran reviewed (approved):
Looks good, just one fix to make to pass CI

markdurrant committed:
Update external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json

Co-authored-by: Tak Tran <contact@tutaktran.com>

JamesSwinton commented on external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json:
```diff
@@ -0,0 +1,162 @@
+            {
+                "title": "Chart Essentials",
+                "description": "Basic chart types, including line, bar, column, and pie charts."
+            },
```
Should we add a link to Gallery here? https://ag-grid.com/charts/gallery/

JamesSwinton commented on external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json:
```diff
@@ -0,0 +1,162 @@
+            {
+                "title": "Advanced Chart Types",
+                "description": "Access to 20+ additional chart types, including Range, Heatmap, Sankey and more."
+            },
```
Should we add a link to the gallery here aswell? https://ag-grid.com/charts/gallery/#maps

JamesSwinton commented on external/ag-website-shared/src/components/features-section/DocsFeaturesSection.json:
```diff
@@ -0,0 +1,162 @@
+            {
+                "title": "Advanced Interactivity",
+                "description": "Context menus, cross-hairs, navigator, zoom and chart synchronization."
+            },
```
Link to Context menu from here? https://ag-grid.com/charts/javascript/context-menu/

markdurrant committed:
AG-3390 - Update features links

markdurrant committed:
git subrepo push external/ag-website-shared

subrepo:
  subdir:   "external/ag-website-shared"
  merged:   "9bf7cc3e84"
upstream:
  origin:   "git@github.com:ag-grid/ag-website-shared.git"
  branch:   "latest"
  commit:   "9bf7cc3e84"
git-subrepo:
  version:  "0.4.6"
  origin:   "git@github.com:ingydotnet/git-subrepo.git"
  commit:   "73a0129"
